backend/app/api/routers/cipher.py

```
@@ -95,8 +95,107 @@ def delete_memory(
+            decrypted.append({
+                "pii_type": entry.get("pii_type", "unknown"),
+                "error": f"Missing required fields: {missing}",
+            })
+            continue
+        try:
+            plaintext = decrypt_pii_field(entry, passphrase)
+            decrypted.append({
+                "pii_type": entry.get("pii_type", "unknown"),
+                "original": plaintext,
+            })
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)